src/gateway/models.py

```diff
@@ -63,11 +64,55 @@ def _set_fk_pragma(dbapi_connection, connection_record):
 session_factory = sessionmaker(autocommit=False, autoflush=False, bind=engine)
 Session = scoped_session(session_factory)
 
+import traceback as tb
+import threading
+
+
+def print_stacktrace(stacktrace):
```

Comment:
Make this a private function
```suggestion
def _print_stacktrace(stacktrace):
```